### ISSUE TITLE
Execution failed for task ':app:generatePackageList' Issue with @react-native-community/cli-platform-android/native_modules.gradle' line: 122

### ISSUE THREAD

**Issue by AbdulBari5** [react-native-community/cli#1905]
## Description
System:
  OS: Linux 5.15 Ubuntu 20.04.6 LTS (Focal Fossa)
  CPU: (4) x64 Intel(R) Core(TM) i5-6300U CPU @ 2.40GHz
  Memory: 8.41 GB / 23.36 GB
  Shell:
    version: 5.0.17
    path: /bin/bash
Binaries:
  Node:
    version: 16.15.0
    path: ~/.nvm/versions/node/v16.15.0/bin/node
  Yarn:
    version: 1.22.19
    path: ~/.nvm/versions/node/v16.15.0/bin/yarn
  npm:
    version: 8.5.5
    path: ~/.nvm/versions/node/v16.15.0/bin/npm
  Watchman: Not Found
SDKs:
  Android SDK:
    API Levels:
      - "23"
      - "30"
      - "31"
      - "32"
      - "33"
    Build Tools:
      - 30.0.2
      - 30.0.3
      - 31.0.0
      - 33.0.0
    System Images:
      - android-29 | Google APIs Intel x86 Atom
      - android-33 | Google APIs Intel x86 Atom_64
    Android NDK: Not Found
IDEs:
  Android Studio: Not Found
Languages:
  Java:
    version: 17.0.6
    path: /usr/bin/javac
  Ruby: Not Found
npmPackages:
  "@react-native-community/cli":
    installed: 11.1.2
    wanted: ^11.1.2
  react:
    installed: 18.2.0
    wanted: 18.2.0
  react-native:
    installed: 0.71.3
    wanted: 0.71.3
npmGlobalPackages:
  "*react-native*": Not Found
Android:
  hermesEnabled: false
  newArchEnabled: false
iOS:
  hermesEnabled: Not found
  newArchEnabled: Not found

The issue is located on line 122 of native_modules.gradle, This issue is reported on 
https://stackoverflow.com/questions/67093053/react-native-execution-failed-for-task-appgeneratepackagelist/72178679#72178679
The solutions aims to downgrade gradle versions but it causes more issues as gradle 6.9 isn't supported by new architecturee of react-native. 
The solution is to mentioned here and it works. 
https://stackoverflow.com/questions/67093053/react-native-execution-failed-for-task-appgeneratepackagelist/72178679#72178679
The problem I am facing is that when building the app through eas or expo, the same error comes because i cannot manually remove those lines from the package at the time of build, so it would be good if there's a way to solve it in the package itself.

**Comment by cortinico** [comment]
> The issue is located on line 122 of native_modules.gradle, This issue is reported on 

What is the issue you're referring to?

**Comment by EmanoelLoreto** [comment]
Has anyone managed to fix the problem?

**Comment by adamTrz** [comment]
Hi @AbdulBari5.  The issue you've mentioned is two years old and have been fixed already. 
Could you please provide us with some more info and preferably Reproducible Demo that we can check out?

**Comment by shahanahmed86** [comment]
Did anyone find solution to that?

**Comment by taha200** [comment]
https://stackoverflow.com/a/72178679
This link is solved this problem
node_modules/@react-native-community/cli-platform-android/native_modules.gradle

When you are trying to install any package you need to do this

**Comment by shahanahmed86** [comment]
Thanks, I've found a package called [`"patch-package": "^8.0.0"`](https://github.com/ds300/patch-package). So what I did was to [change](https://stackoverflow.com/a/72178679) inside of the `node_module` and run the `patch-package` with the argument of the package name. And then write a `postInstall` script inside of package.json with the value of `patch-package` and voila the fix is now permanent.
